From 13fb01d8155207f34dcfbf36d51faf93506c72fe Mon Sep 17 00:00:00 2001
From: Johannes Berg <johannes.berg@intel.com>
Date: Fri, 6 Nov 2015 11:26:26 +0100
Subject: [PATCH] iwlwifi: pcie: remove ICT allocation message
Git-commit: 13fb01d8155207f34dcfbf36d51faf93506c72fe
Patch-mainline: 4.5-rc1
References: FATE#321353, FATE#323335

This message isn't very useful and presents a security risk
due to the use of %p - remove it.

Signed-off-by: Johannes Berg <johannes.berg@intel.com>
Signed-off-by: Emmanuel Grumbach <emmanuel.grumbach@intel.com>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/net/wireless/iwlwifi/pcie/rx.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/net/wireless/iwlwifi/pcie/rx.c
+++ b/drivers/net/wireless/iwlwifi/pcie/rx.c
@@ -1491,10 +1491,6 @@ int iwl_pcie_alloc_ict(struct iwl_trans
 		return -EINVAL;
 	}
 
-	IWL_DEBUG_ISR(trans, "ict dma addr %Lx ict vir addr %p\n",
-		      (unsigned long long)trans_pcie->ict_tbl_dma,
-		      trans_pcie->ict_tbl);
-
 	return 0;
 }
 
